osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/VerifyImportedScoresCommand.cs

```diff
@@ -39,11 +49,14 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
             elasticQueueProcessor = new ElasticQueuePusher();
             Console.WriteLine($"Indexing to elasticsearch queue(s) {elasticQueueProcessor.ActiveQueues}");
 
+            if (DryRun)
+                Console.WriteLine("RUNNING IN DRY RUN MODE.");
+
             using var conn = DatabaseAccess.GetConnection();
 
             while (!cancellationToken.IsCancellationRequested)
             {
-                List<ElasticQueuePusher.ElasticScoreItem> elasticItems = new List<ElasticQueuePusher.ElasticScoreItem>();
+                HashSet<ElasticQueuePusher.ElasticScoreItem> elasticItems = new HashSet<ElasticQueuePusher.ElasticScoreItem>();
```

Comment:
Making this a `HashSet<>` seems dubious given that `ElasticScoreItem` is a plain class with no equality members overridden. So if the intent here was to deduplicate anything, then I'm pretty sure that will not happen, because the default comparison method for a plain class without equality members overridden is *reference* equality.

Reply:
Yeah it kinda was for deduping, but it's not a huge deal for now.

I'll change the logic to dedupe because it's simple to do and probably reads better.

Reply:
In fact, I think there was a missing index op for the new cases I added, so I'll need to run manual reindexing.

Reply:
> I'll change the logic to dedupe

I got a re-review request, but nothing seems to have happened in this respect (yet?)

Reply:
https://github.com/peppy/osu-queue-score-statistics/commit/5ba0eda4ef638ac9841f39bb9e0ef3b7459094c9 is intended to "dedupe", aka move the index request to the `finally` block.